<def f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h' l='748' type='void lldb_private::python::PythonScript::PythonScript(const char * script)'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='886' u='c' c='_ZNK12lldb_private6python14PythonCallable10GetArgInfoEv'/>
<use f='llvm/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp' l='1090' u='c' c='_ZNK12lldb_private6python15PythonException13ReadBacktraceB5cxx11Ev'/>
<use f='llvm/lldb/unittests/ScriptInterpreter/Python/PythonDataObjectsTests.cpp' l='792' u='c' c='_ZN37PythonDataObjectsTest_TestScript_Test8TestBodyEv'/>
<use f='llvm/lldb/unittests/ScriptInterpreter/Python/PythonDataObjectsTests.cpp' l='809' u='c' c='_ZN41PythonDataObjectsTest_TestExceptions_Test8TestBodyEv'/>
<use f='llvm/lldb/unittests/ScriptInterpreter/Python/PythonDataObjectsTests.cpp' l='829' u='c' c='_ZN41PythonDataObjectsTest_TestExceptions_Test8TestBodyEv'/>
